lib/wallet/src/Cardano/Wallet/DB/Layer.hs

```diff
@@ -590,15 +601,19 @@ newDBLayerWith _cacheBehavior _tr ti SqliteContext{runQuery} = do
                         let
                             delta = Just GarbageCollectTxWalletsHistory
                         in  (delta, Right ())
-        , listWallets = map unWalletKey <$> selectKeysList [] [Asc WalId]
+
+        , listWallets_ = map unWalletKey <$> selectKeysList [] [Asc WalId]
```

Comment:
I am recruiting people into my church of `fmap . fmap` aka `<<$>>`, ⛪

Reply:
And what about `<<<$>>>`?